src/test/resources/rest-api-spec/test/fstore/80_search.yaml

```diff
@@ -0,0 +1,120 @@
+---
+setup:
```

Comment:
nitpick: the test name could be bit more explicit, e.g. setup simple test models with a test indice